test/conservation_checker_tests.jl

```diff
@@ -41,7 +42,10 @@ for FT in (Float32, Float64)
         space = CC.CommonSpaces.CubedSphereSpace(FT; radius = FT(6371e3), n_quad_points = 4, h_elem = 4)
 
         # set up model simulations
-        atmos = TestAtmos((; space = space))
+        initial_energy = ones(space) .* CC.Spaces.undertype(space)(1e6)
+        initial_water = ones(space) .* CC.Spaces.undertype(space)(1e5)
```

Comment:
```suggestion
        initial_energy = ones(space) .* 1e6
        initial_water = ones(space) .* 1e5
```
I think the float type will be inherited from the space

Reply:
1e6 is Float64 and `ones()` can be Float32, which leads to a Float64 Field, and this cannot be used in other operations below because of the mixed type